Duckling/Time/CA/Rules.hs

```diff
@@ -152,8 +152,8 @@ ruleNit = Rule
     [ regex "nit"
     ]
   , prod = \_ ->
-      let from = hour False 22
-          to = hour False 1
+      let from = hour False 20
```

Comment:
why did this change?

Reply:
"nit" means night. Being CA a language that has both evening and night, I think 20-24 represents a better time range for it.